bundles/action/org.openhab.action.pushover/README.md

```diff
@@ -40,13 +40,39 @@ For specific information on each item, see the [Pushover API](https://pushover.n
 - `pushover(String apiToken, String userKey, String message, String device, int priority)`
 - `pushover(String apiToken, String userKey, String message, String device, String title, String url, String urlTitle, int priority, String soundFile)`
 
+- `sendPushoverEmergency(String message)`
+- `sendPushoverEmergency(String message, String title)`
+- `sendPushoverEmergency(String message, String title, String device)`
+- `sendPushoverEmergency(String message, String title, String device, String soundFile)`
+
+- `sendPushover(String apiToken, String userKey, String message, String device, String title, String url, String urlTitle, int priority, String soundFile)`

```

Comment:
I'd move this one up below all the other `pushover` actions.
I'm not sure if this method does much good. What's your idea behind it?

Reply:
No, I should remove it for now. This action is a duplicate of the original most parameterized pushover action but uses a different technique for sending the notifications and parsing the response. The binding currently uses the XML API of Pushover. I'm switching it to JSON.